src/Helpers/ProjectHelpers.cs

```diff
@@ -129,7 +130,7 @@ public static ProjectItem AddFileToProject(this Project project, FileInfo file,
 				return null;
 			}
 
-			ProjectItem item = project.ProjectItems.AddFromFile(file.FullName);
+			ProjectItem item = project.ProjectItems.AddFromTemplate(file.FullName, file.FullName);
```

Comment:
I think this should fix it
```suggestion
			ProjectItem folderItem = _dte.Solution.FindProjectItem(file.DirectoryName);
			ProjectItem item = folderItem.ProjectItems.AddFromTemplate(file.FullName, file.Name);
```

Reply:
I tried this, but if you use the extension and add "Bar\Baz\Class1.cs" (thus creating new folders) those aren't yet project items so `folderItem` in this would be null.  Need to AddFolder() on the DirectoryName first (which I think should work and hopefully fail gracefully if exists)